src/site/stages/build/drupal/graphql/nodeBasicLandingPage.graphql.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * The alerts that appear above content.
+ */
+
+module.exports = `
+fragment nodeBasicLandingPage on NodeBasicLandingPage {
+  title
+  created
+  fieldContentBlock {
```

Comment:
I think you're getting that error because it is missing the `entityBundle`. I think you need to import and include `${entityElementsFromPages}` in the query

Reply:
ah perfect, thanks! 🙂

Reply:
updated!